openupgrade_scripts/scripts/account/14.0.1.1/post-migration.py

```diff
@@ -324,6 +326,16 @@ def post_statements_with_unreconciled_lines(env):
             AND bstl.is_reconciled IS DISTINCT FROM TRUE
         """,
     )
+    # New statements with reconciled items
+    openupgrade.logged_query(
+        env.cr,
+        """SELECT bst.id FROM account_bank_statement bst
```

Comment:
```suggestion
        """SELECT DISTINCT bst.id FROM account_bank_statement bst
```

Reply:
Not needed, as I deduplicate later doing a Python set.

Reply:
Oh, right